fr_FR/docs/recipes/react.md

```diff
@@ -0,0 +1,151 @@
+___
+**Note du traducteur**
+
+C'est la traduction du fichier [react.md](https://github.com/sindresorhus/ava/blob/master/docs/recipes/react.md). Voici un [lien](https://github.com/sindresorhus/ava/compare/d3f18233c0f1f0bf351dc1dd7aae9c93aecb2da6...master) vers les différences avec le master de AVA (Si en cliquant sur le lien, vous ne trouvez pas le fichier `babelrc.md` parmi les fichiers modifiés, vous pouvez donc en déduire que la traduction est à jour).
+___
+
+# Tester les composants React
+
+## Installer Babel
+
+La première étape est de configurer Babel pour transformer le code JSX des tests en ajoutant une section AVA à votre fichier package.json:
+
+```json
+{
+  "ava": {
+    "require": ["babel-register"]
+  },
+  "babel": {
+    "presets": ["react"]
+  }
+}
+```
+
+Vous pouvez trouver plus d'information sur la configuration de Babel avec AVA dans la [recette sur babelrc](/fr_FR/docs/recipes/babelrc.md).
+
+## Utiliser [Enzyme](https://github.com/airbnb/enzyme)
+
+Commençons par voir comment utiliser AVA avec l'une des bibliothèques(?)(libraries) de test React parmi les plus populaires : [Enzyme](https://github.com/airbnb/enzyme).
```

Comment:
Not sure about how to translate "library", if I was speaking in french I would probably say "librairie" because I'm so used to read it in english but I guess the correct french translation is "bibliothèque", not sure what we want in AVA docs.